src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx

```diff
@@ -272,7 +272,7 @@ export default class ComparisonGroupManager extends React.Component<
                         this.props.store.updateComparisonGroupsFilter();
                     }}
                 >
-                    View
+                    Filter
```

Comment:
I thought it was "Select".   I can see "filter" as well, but maybe "Apply to filter" ???  @jjgao can you make an executive decision here?

Reply:
And do change the comment above and the name of the getter accordingly